test/integration/functional_test_tunnel_test.go

```diff
@@ -131,6 +133,49 @@ func validateTunnelStart(ctx context.Context, t *testing.T, profile string) {
 	tunnelSession = *ss
 }
 
+// validateNoSecondTunnel ensures only 1 tunnel can run simultaneously
+func validateNoSecondTunnel(ctx context.Context, t *testing.T, profile string) {
+	checkRoutePassword(t)
+
+	exitCodeCh := make(chan int)
+	sessions := make([]*StartSession, 2)
+
+	var runTunnel = func(idx int) {
+		args := []string{"-p", profile, "tunnel", "--alsologtostderr"}
+
+		ctx2, cancel := context.WithTimeout(ctx, Seconds(5))
+		session, err := Start(t, exec.CommandContext(ctx2, Target(), args...))
+		if err != nil {
+			t.Errorf("Failed to start tunnel for the first time")
+		}
+		sessions[idx] = session
+		defer cancel()
+
+		err = session.cmd.Wait()
+		if exErr, ok := err.(*exec.ExitError); ok {
+			exitCodeCh <- exErr.ExitCode()
+		} else {
+			exitCodeCh <- -1
+			t.Errorf("Second tunnel command failed due to a different reason: %s", err)
```

Comment:
```suggestion
			t.Errorf("tunnel command failed due to a different reason: %s", err)
```
This error can be misleading, if the first tunnel fails to run for some reason, the error message `Second tunnel command failed due to a different reason` can confuse the user as they're going to assume it's the second tunnel when it can be the first tunnel that failed.

Reply:
Fixed.